erpnext/public/js/shopping_cart.js

```diff
@@ -5,14 +5,66 @@
 frappe.provide("erpnext.shopping_cart");
 var shopping_cart = erpnext.shopping_cart;
 
+var setCookie = function (cname, cvalue, exdays) {
```

Comment:
functions already exist in frappe. please check

Reply:
Reused existing frappe cookies function